set.mm

```diff
@@ -510554,6 +510819,1209 @@ commutative monoid (=vectors) together with a semiring (=scalars) and a
+          VVPVWENOVXSVYNWUPXGXGZVXFBFHVWSVWTNOVXSWVTWYAXGVXFBFHVWSVXANOVXSWVTWY
+          BXGXMVXFBFHWXLVUCNOVUDVXSWYCYAYBYDXKVXFBCFVUCVUCNPVUDVXPVXFVXNVUCBVRV
+          XPBFVUCNVUDUVMWHUVPYDYFYFUVQVWRUNBEFGHVVEVUBVVFWFVUCNRVXJVXLVUDOVUEVV
+          JVVGVWPVWNXSYGYHVVKUMBEFGHVVBVUBVVCWFVUCNRVXIVXKVUDOVUEVYAYGYHVVKVVHV
+          VBVVEUXGVDZVVTVVKUYLUXGVVBVVEAUYLUXGWCVVDVVGAUYLUXGUYKUYKVAWEVUPVUQVU
+          RUVRUXHWKUXKUYKUWOUXHUXJUVSVUSWMAUWNWEVRZWYEUXGWEVRVUTVUTUFUGUWNUWNUX
+          FWEWEUVTUWAUYLWGZWOZWRZYIVVKVXTVYEVVTWEVRZWYDVVTWCVYBVYGWYIVVKVVRVVSY
+          MWNUFUGVVBVVEUWNUWNUXFVVTUXGWEUFUKYJZUGULYJZVSZUXDVVRUXEVVSWYLUWTVVNU
+          XCVVQCWYLUWQVVLUWSVVMHWYLUWPVVBVBWYJWYKYKZYLWYLUWRVVEVCWYJWYKXCZYLZXK
+          WYLUXAVVOUXBVVPHWYLUWRVVEVBWYNYLWYLUWPVVBVCWYMYLZXKXKWYLUXBVVPUWSVVMH
+          WYPWYOXKYNVUGYOXQYDVVKVVIVVCVVFUXGVDZVWIVVKUYLUXGVVCVVFWYHYIVVKVYPWUB
+          VWIWEVRZWYQVWIWCZVYRWUDWYRVVKVWGVWHYMZWNUFUGVVCVVFUWNUWNUXFVWIUXGWEUF
+          UJYJZUGUIYJZVSZUXDVWGUXEVWHXUCUWTVWCUXCVWFCXUCUWQVWAUWSVWBHXUCUWPVVCV
+          BXUAXUBYKZYLXUCUWRVVFVCXUAXUBXCZYLZXKXUCUXAVWDUXBVWEHXUCUWRVVFVBXUEYL
+          XUCUWPVVCVCXUDYLZXKXKXUCUXBVWEUWSVWBHXUGXUFXKYNVUGYOZXQYDUWDUWBAVYPWU
+          BVVIUWNVRAVYPVSZWUBVSZVVIWYQUWNAVVIWYQWCVYPWUBAUYLUXGVVCVVFWYGYIWRXUJ
+          WYQVWIUWNXUJVYPWUBWYRWYSAVYPWUBWTZXUIWUBXCZWYRXUJWYTWNXUHXQXUJVWGVWHB
+          GXUJBCFVWCVWFNPAVXNVYPWUBVXOWRXUJBFHVWAVWBNOAVXQVYPWUBVXRWRZXUJVYPVYQ
+          XUKVYTWHXUJGBVWBAVUKVYPWUBVUNWRZXUJWUBWUCXULWUFWHZXFXGXUJBFHVWDVWENOX
+          UMXUJWUBWUJXULWUKWHXUJGBVWEXUNXUJVYPWUMXUKWUNWHZXFXGXHXUJHGVUHVWEVWBW
+          URAVUJVYPWUBTWRXUPXUOXJWDYPYPUWCWYFUEUWEUWFAUYMUYSEAUFUGUWKUWLUWNUWNU
+          XFUYSUYLWEWYGAUWPUWKWCZUWRUWLWCZVSZVSZUXDUYQUXEUYRXUTUWTUYOUXCUYPCXUT
+          UWQIUWSLHXUTUWQUWKVBUSZIXUTUWPUWKVBAXUQXURUWGZYLAIBVRZXUSKGVRZXVAIWCU
+          AAXVDXUSUCYQZIKBGYRYSYDXUTUWSUWLVCUSZLXUTUWRUWLVCAXUQXURUWHZYLAJBVRZX
+          USLGVRZXVFLWCUBAXVIXUSUDYQZJLBGYTYSYDZXKXUTUXAJUXBKHXUTUXAUWLVBUSZJXU
+          TUWRUWLVBXVGYLAXVHXUSXVIXVLJWCUBXVJJLBGYRYSYDXUTUXBUWKVCUSZKXUTUWPUWK
+          VCXVBYLAXVCXUSXVDXVMKWCUAXVEIKBGYTYSYDZXKXKXUTUXBKUWSLHXVNXVKXKYNUYTV
+          UAUYSWEVRAUYQUYRYMWNUWIUWJYD $.
+      $}
+
+      ${
+        $d .(x) p q u v $.  $d .+ a b p q $.  $d .x. a b k $.  $d .x. f g $.
+        $d .x. p q u $.  $d .x. t $.  $d .~ f g p q u v $.  $d B a b $.
+        $d B f g $.  $d B k p q $.  $d B u v $.  $d E a b $.  $d E p q $.
+        $d F a b $.  $d F p q $.  $d G a b $.  $d G p q $.  $d H a b $.
+        $d H p q $.  $d R a b t $.  $d R f g $.  $d R k $.  $d R p q $.
+        $d R t u $.  $d S a b $.  $d S f g p q u v $.  $d S k $.  $d S t $.
+        $d a b ph u v $.  $d f g ph $.  $d p ph q $.
+        rlocmulval.1 $e |- .(x) = ( .r ` L ) $.
+        $( Value of the addition in the ring localization, given two
+           representents.  (Contributed by Thierry Arnoux, 4-May-2025.) $)
```

Comment:
"Representatives" here as well.

Reply:
Fixed in https://github.com/metamath/set.mm/pull/4810/commits/f635662cfffd84458584f2c4a905b88f32b45a51